bundles/catalogue/metadatasearch/view/MetadataSearchContainer.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { METADATA_BUNDLE_LOCALIZATION_ID } from '../instance';
+import { SearchInput, Spin } from 'oskari-ui';
+import { AdvancedSearchContainer } from './advanced-search/AdvancedSearchContainer';
+import { MetadataSearchResultListContainer } from './resultlist/MetadataSearchResultListContainer';
+import { FlexRowCentered } from './advanced-search/AdvancedSearchStyledComponents';
+
+const Description = () => {
+    return <div>{Oskari.getMsg(METADATA_BUNDLE_LOCALIZATION_ID, 'metadataSearchDescription')}</div>;
```

Comment:
We could use `<Message />` here, but since this is the only instance to use it AND we need to use Oskari.getMsg() anyway for the placeholder, I think it's ok to not use the Message-tag here.

Reply:
Changed this. I agree it's cleaner that way even if we still gotta use the getMsg later on.